CDCgov/prime-simplereport
# Add identifier and issued date for diagnosticReport

# BACKEND PULL REQUEST

## Related Issue
Resolves #5429 

## Changes Proposed
- adds `identifier` to `specimen` and `diagnosticReport`
- adds `issued` to `diagnosticReport`

## Additional Information

## Testing
Available on `dev` for testing
- In Azure, do the following:
  - Go to "Storage accounts"
  - Click "simplereportdevapp"
  - Click "Queues" on the sidebar
  - Click on "fhir-data-publishing"
- Go to the app, do the following:
  - Submit a multiplex test
- Go back to Azure, and do the following:
  - Open up the message in the queue and ensure the new values have been added and are correct

Diagnostic Report:
<img width="366" alt="Screen Shot 2023-03-14 at 13 47 42" src="https://user-images.githubusercontent.com/20211771/225112163-2d0efe8c-ebe9-46ca-8e8f-466a7ecc08fb.png">

Specimen:
<img width="407" alt="Screen Shot 2023-03-14 at 13 47 13" src="https://user-images.githubusercontent.com/20211771/225112374-ef63050b-026f-479f-80f4-eb37ac7c1763.png">
